packages/v4/patternfly-docs/content/design-guidelines/components/time picker/time-picker.md

```diff
@@ -38,13 +38,13 @@ Until the user selects a start time, the end time field will be disabled.
 ### Date and time range selection
 Combine a date picker and time picker when you want users to select a range of times across multiple days. 
 
-For more information about using date and time pickers together, visit our [date picker guidelines](/components/date-picker/design-guidelines).
+For more information about using date and time pickers together, visit our [date picker guidelines](/components/date-and-time/date-picker).
 
 ## Variations
 ### Time picker in a toolbar
 Use time pickers as filters in a toolbar. Note that time pickers aren't commonly used on their own, and will usually be accompanied by a [date picker](/components/date-picker).
 
-See the [date picker usage guidelines](/components/date-picker/design-guidelines) to learn more about using date pickers and time pickers in a toolbar. 
+See the [date picker usage guidelines](/components/date-and-time/date-picker) to learn more about using date pickers and time pickers in a toolbar. 
```

Comment:
Same here. This url should be `I think this url should be /components/date-and-time/date-picker/design-guidelines`